Dockerfile

```diff
@@ -8,11 +8,14 @@ LABEL "com.github.actions.color"="red"
 LABEL "repository"="https://github.com/Ilshidur/actions"
 LABEL "homepage"="https://github.com/Ilshidur/action-slack"
 LABEL "maintainer"="Ilshidur <ilshidur@gmail.com>"
-LABEL "version"="1.6.2"
+LABEL "version"="2.0.0"
 
-ADD package.json package-lock.json /
-RUN npm ci --production
-ADD entrypoint.js /
+ADD package.json yarn.lock /
+RUN yarn --production
+COPY . /
 RUN chmod +x /entrypoint.js
 
+ARG GITHUB_EVENT_PATH=./event-example.json
```

Comment:
I don't understand why you are suggesting to define this env var at build time.
GitHub already defines it when running the container. Am I missing something ?

Reply:
used this technique to set a default GITHUB_EVENT_PATH to the container when it isn't set.
_aka: when running the container locally to test, not have to set it all the time._
> ye, im that lazy.. ahahah

Reply:
Let's keep it then ;-)

Reply:
Let's keep it then ;-)